src/Resources/views/admin_order/detail.html.twig

```diff
@@ -377,7 +373,7 @@
                             <!-- Body -->
                             <div class="card-body ">
                                 <div class="media ng-scope">
```

Comment:
List is somehow bit messed-up.
<img width="404" alt="image" src="https://user-images.githubusercontent.com/11687066/231187342-6a0653c1-5ccf-484a-bd3a-f4c79de69eff.png">


```suggestion
                            <div class="card-body  p-3">
                                <div class="media ng-scope d-flex align-items-start">
```